packages/appstore/src/types/common.types.ts

```diff
@@ -183,3 +183,17 @@ export type TLinkedTo = {
     balance?: string;
     currency?: string;
 };
+
+export type TWalletAccount = {
```

Comment:
@sergei-deriv You have defined this type 3 times in this PR and they are sightly different, I believe creating a `useWalletAccounts` hook will resolve all of that since we can infere the types.

Reply:
@farzin-deriv thank you, I think it makes sense to move it to hooks. Thanks a lot, I wanted to hear your opinion about `wallet_accounts` in store.